hoptimator-cli/src/main/java/com/linkedin/hoptimator/HoptimatorCliApp.java

```diff
@@ -264,6 +266,97 @@ public boolean echoToFile() {
     }
   }
 
+  private class InsertCommandHandler implements CommandHandler {
+
+    @Override
+    public String getName() {
+      return "insert";
+    }
+
+    @Override
+    public List<String> getNames() {
+      return Collections.singletonList(getName());
+    }
+
+    @Override
+    public String getHelpText() {
+      return "Run an ephemeral pipeline with an existing sink.";
+    }
+
+    @Override
+    public String matches(String line) {
+      String sql = line;
+      if (sql.startsWith(SqlLine.COMMAND_PREFIX)) {
+        sql = sql.substring(1);
+      }
+
+      if (sql.startsWith("insert into")) {
+        sql = sql.substring("insert into".length() + 1);
+        return sql;
+      }
+
+      return null;
+    }
+
+    @Override
+    public void execute(String line, DispatchCallback dispatchCallback) {
+      String sql = line;
+      if (sql.startsWith(SqlLine.COMMAND_PREFIX)) {
+        sql = sql.substring(1);
+      }
+
+      if (sql.startsWith("insert into")) {
+        sql = sql.substring("insert into".length() + 1);
+      }
+
+      String connectionUrl = sqlline.getConnectionMetadata().getUrl();
+      try {
+        String[] parts = sql.split("[sS][eE][lL][eE][cC][tT]");
```

Comment:
nit: This could be case-insensitive regex. I don't have strong preferences for readability either way though.
```java
jshell> String stmt = "SelEcT * FROM RAWKAFKA.\"test-sink\" LIMIT 5;"
stmt ==> "SelEcT * FROM RAWKAFKA.\"test-sink\" LIMIT 5;"

jshell> stmt.split("(?i)select")
$7 ==> String[2] { "", " * FROM RAWKAFKA.\"test-sink\" LIMIT 5;" }
```

Reply:
Neat! Will fix.